test/dump/relocations-long-func-bodies.txt

```diff
@@ -0,0 +1,98 @@
+;;; TOOL: run-objdump
+;;; ARGS0: -r
+;;; ARGS1:
+(module
+  (global $g i32 (i32.const 0))
+  (func $long_func
```

Comment:
Can we have a preceding function there that also contains a relocation so check that the relocations in preceding functions are not also adjusted?

Reply:
sure, I'll add one